main.go

```diff
@@ -35,12 +40,30 @@ func main() {
 		os.Exit(1)
 	}
 
-	encoder := json.NewEncoder(os.Stdout)
+	if saveOutput(channel, *targetFile); err != nil {
+		fmt.Println("Error saving output:", err)
+		os.Exit(1)
+	}
+}
+
+func saveOutput(channel Channel, targetFile string) error {
+	var encoder *json.Encoder
+	if targetFile == stdOutOutput {
+		encoder = json.NewEncoder(os.Stdout)
+	} else {
+		file, err := os.Create(targetFile)
```

Comment:
It assumes that the target directory exists. 

It's also fine if it's not modified.

Reply:
I believe this is fine for the use case we have here, especially given the limited usage of this script.

Let's leave this as is.